apps/air-discount-scheme/backend/src/app/modules/flight/flight.service.ts

```diff
@@ -36,6 +36,10 @@ const availableFinancialStates = [
   financialStateMachine.states[States.sentDebit].key,
 ]
 
+export const CONNECTING_FLIGHT_GRACE_PERIOD = 48 * (1000 * 60 * 60) // 48 hours in milliseconds
+export const REYKJAVIK_FLIGHT_CODES = ['RVK', 'REK']
+export const AKUREYRI_FLIGHT_CODES = ['AK']
+
```

Comment:
One thing we need to add here is `ALLOWED_CONNECTING_FLIGHT_CODES = ['VPN', 'GRY', 'THO']`

Connecting flights are only allowed to these places